tests/assets/move-projects/car-wash-example/sources/CarWash.move

```diff
@@ -0,0 +1,92 @@
+/// This module defines a minimal Coin and Balance.
```

Comment:
Let's mention that this is just an example and it's not perfect - "use it only for testing purposes"

Reply:
Modified it :blush: